examples/chapter-form/index.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import { Outlet, Route } from 'react-router-dom'
+import FormRouter from '../../src/routing/Router';
+import Chapter from '../../src/routing/Chapter';
+import Page from '../../src/routing/Page';
+import IntroductionPage from '../../src/form-layout/IntroductionPage'
+import { Link } from 'react-router-dom';
+
+const NoMatch = (props) => (
+  <main style={{ padding: '1rem' }}>
+    <p>There is nothing here! {props.name}</p>
+  </main>
+);
+
+const FormIntroductionPageChapter = () => {
+  return (
+    <>
+      <Page title="Chapter Form" nextPage="chapter-one">
+        <p> Chapter Form </p>
+      </Page>
+    </>
+  )
+}
+
+const ChapterOne = (props) => (
+  <>
+    <h2> Chapter One</h2>
+    <Link to="page-one">PageOne</Link>
+
+    <Chapter>
+      <p>Custom UI content that can go inside chapter 1</p>
+    </Chapter>
+  </>
+);
+
+const ChapterOnePageOne = (props) => (
+  <Page nextPage="../page-two">
+    <p>chapter one, page one</p>
+  </Page>
+);
+
+const ChapterOnePageTwo = (props) => (
+  <Page nextPage="../../chapter-two">
```

Comment:
to add on to this we might have to think about using full paths (ex. /my-form/chapter-one/page-one) because with react router v6 to change to another chapter with another outlet you will have to define the root and following paths after.

Reply:
@micahchiang @bjmccotter7192 okay I updated this branch so that the URLs are absolute. I will work to put tests in this branch then publish the PR. Thanks for the feedback!